src/lib/vocab/tie/tie.schema.ts

```diff
@@ -1,6 +1,11 @@
 export const TieSchema = {
 	$id: '/schemas/Tie.json',
 	type: 'object',
+	description: `
+		Ties are part of the Entity/Tie graph system.
+		They represent the relationship between two entities and name the relationship.
+		The relationship descrbied follows the <source> has relationship of <type> with <dest>.
```

Comment:
```suggestion
		A Tie specifies "the [source] has relationship of [type] with [dest]."
```

I think it's likely we'll switch to HTML for docs so we can get links (or Fuz) so best to avoid angle brackets.